lanl/docker-compose/ochami-services-noauth.yml

```diff
@@ -146,8 +146,10 @@ services:
     container_name: dnsmasq-dhcp-noauth
     hostname: dnsmasq-dhcp-noauth
     environment:
-      - smd_endpoint=172.16.0.253
-      - bss_endpoint=172.16.0.253
+      - smd_endpoint=localhost
```

Comment:
could you use the local container names for these? they should resolve as dns names

Reply:
They don't resolve as `network` and `network_mode: "host"` cannot be used at the same time. The dnsmasq-dhcp container is not in the internal or external networks.